mappings/servers/akumamc/metadata.json

```diff
@@ -0,0 +1,35 @@
+{
+    "id": "akumamc",
+    "name": "AkumaMC",
+    "description": "AkumaMC is the best Minecraft OP Prison & Skyblock Server out there. Your experience is our #1 priority.",
```

Comment:
Your description is too long
Needs to be a max of 50 characters as per the [docs](https://docs.cheatbreaker.net/server-mappings/adding-servers/metadata#basic)